config/routes.rb

```diff
@@ -17,16 +17,21 @@
     resources :tickets, only: %i[index show]
   end
 
-  resources :ticket_desks, only: %i[index show] do
-    resources :movies, only: %i[index show] do
-      resources :screenings, only: %i[index show] do
-        resources :reservations do
-          resources :tickets, only: %i[index show]
-        end
-      end
+  #offline endpoint
+  scope module: 'offline' do
+    resources :ticket_desks, only: %i[index show] do
+      resources :reservations
     end
   end
 
+  #online endpoint
+  #namespace :users do
```

Comment:
Here you can also use `scope module` :) Of course, it's up to you, both ways are good.